setup.py

```diff
@@ -99,14 +98,14 @@ def finalize_options(self):
             sys.exit(-1)
 
 
-class Cleaner(clean):
+class Cleaner(Command):
```

Comment:
Do we know exactly what we lose here when not inheriting from `clean`?

Reply:
Not yet, but the outputs for "python setup.py clean" command doesn't change with this modification.

Reply:
Well, I just notice that when running tox locally the directory "docs/developer/kytos_api/" were left behind. Once tox were running "python setup.py ci", I added the command "python setup.py clean" to ci command to resolve it.